### PR TITLE
Fix Documentation for Client Pass Through Configuration

### PR DESCRIPTION
**Pull request by michaeljmarshall**
### Motivation

Fix table formatting and copy docs to 2.10.1.

- [x] `doc`

### CODE REVIEW

File: site2/website/versioned_docs/version-2.10.1/reference-configuration.md [hunk 1]
```diff
@@ -378,6 +378,16 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
 | strictBookieAffinityEnabled | Enable or disable the strict bookie isolation strategy. If enabled, <br /> - `bookie-ensemble` first tries to choose bookies that belong to a namespace's affinity group. If the number of bookies is not enough, then the rest bookies are chosen. <br /> - If namespace has no affinity group, `bookie-ensemble` only chooses bookies that belong to no region. If the number of bookies is not enough, `BKNotEnoughBookiesException` is thrown.| false |
 |narExtractionDirectory | The extraction directory of the nar package. <br />Available for Protocol Handler, Additional Servlets, Entry Filter, Offloaders, Broker Interceptor. | System.getProperty("java.io.tmpdir") |
 
+#### Configuration Override For Clients Internal to Broker
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.10.1/reference-configuration.md:381]:
```suggestion
#### Configuration override for clients internal to broker
```
In general, use a lowercase style in text and use sentence-style capitalization for headings.
https://docs.google.com/document/d/1lc5j4RtuLIzlEYCBo97AC8-U_3Erzs_lxpkDuseU0n4/edit#bookmark=id.1oxiv9hq05mn

File: site2/website/versioned_docs/version-2.10.1/reference-configuration.md [hunk 2]
```diff
@@ -378,6 +378,16 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
...
+
+It's possible to configure some clients by using the appropriate prefix.
+
+|Prefix|Description|
+|---|---|
+|brokerClient_| Configure **all** the broker's Pulsar Clients and Pulsar Admin Clients. These configurations are applied after hard coded configuration and before the above brokerClient configurations named above.|
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.10.1/reference-configuration.md:387]:
```suggestion
|brokerClient_| Configure **all** the broker's Pulsar Clients and Pulsar Admin Clients. These configurations are applied after hard coded configuration and before the above broker client configurations named above.|
```
do we need a space between them?

File: site2/website/versioned_docs/version-2.10.1/reference-configuration.md [hunk 3]
```diff
@@ -378,6 +378,16 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
...
+|bookkeeper_| Configure the broker's bookkeeper clients used by managed ledgers and the BookkeeperPackagesStorage bookkeeper client. Takes precedence over most other configuration values.|
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.10.1/reference-configuration.md:388]:
```suggestion
|bookkeeper_| Configure the broker's BookKeeper clients used by managed ledgers and the BookkeeperPackagesStorage bookkeeper client. Takes precedence over most other configuration values.|
```

File: site2/website/versioned_docs/version-2.10.1/reference-configuration.md [hunk 4]
```diff
@@ -378,6 +378,16 @@ brokerServiceCompactionThresholdInBytes|If the estimated backlog size is greater
...
+
+Note: when running the function worker within the broker, these prefixed configurations do not apply to any of those clients. You must instead configure those clients using the `functions_worker.yml`.
```
Comment (Anonymitaet) [inline site2/website/versioned_docs/version-2.10.1/reference-configuration.md:390]:
```suggestion
::: Note

When running the function worker within the broker, these prefixed configurations do not apply to any of those clients. You must configure those clients using the `functions_worker.yml` file.

:::
```
https://docs.google.com/document/d/12De2btkDHQVaqUlHjTqERmroMLKGhHdiC7rEttFTyqc/edit#bookmark=id.jis8plesauvo

**Review by dave2wave — APPROVED** [review]
LGTM

**Comment by michaeljmarshall** [comment]
Thanks @dave2wave

**Pull request merged**